dev-guide/cluster-fleet-evaluation.md

```diff
@@ -146,25 +166,32 @@ the scope and impact of the change and should ideally be done in collaboration
 with your respective OpenShift staff engineer, product manager and PLM contact.
 
 In order of preference, in what teams should choose:
-1. [Insight rule notification](https://access.redhat.com/labs/proactiveissuestracker/) -
+
+1. [Conditional update
+    risks](/enhancements/update/targeted-update-edge-blocking.md).  These appear
+    in-cluster when customers are logged in.  They are mutable, so the advice
+    being provided can be adjusted as new information or improved guidance
+    becomes available.  If we decide that the intended change is too invasive,
+    we can remove the conditional risk declaration entirely.
```

Comment:
how doe the engineering team get the data on this?  Do we report back through telemetry what conditional risks a given cluster is matching?

Reply:
This is the part I am not able to figure out too. I have been reading the docs you linked but nothing is jumping out at me.

Reply:
@wking can answer more fully than I can, but we report the current version, channel, and number of available updates. We could add some canaries and watch for the number of updates to decrease in correlation to that change and figure out then how many clusters match the rule. Too clever for my tastes.

Reply:
part of my confusion was that this is under the "alerting customers" section, not the "Gathering data" section.  So the assumption here is the team already figured out what percent of the fleet will be impacted by their proposed change and they've decided to go ahead with it.

So assuming the team previously gathered the data and said "it looks like if we roll this out, X% of clusters will be blocked from upgrading, and we're ok with that" (and we assume customers will see they are blocked and take the recommended action to unblock their cluster) it seems ok as a tool in the toolbox.  i'm not sure it should be the first one.  I think we'd prefer to get our fleet to the point where 100% of them will be eligible to upgrade, before rolling out the change.

But i do agree with it being in the toolbox.